### ISSUE TITLE
[Bug] InfoWindow.shouldFocus set to false but still getting the focus

### ISSUE THREAD

**Issue by ngothanhtai** [visgl/react-google-maps#274]
### Description

I update to version 0.8.0 to disable the focus feature (shouldFocus) by setting it to false. However, the InfoWindow still receives focus.

FYI, using Google Map API is working fine for me.
```js
const infoWindow = new google.maps.InfoWindow({
  position,
  disableAutoPan: true,
  zIndex: 1,
  pixelOffset: new google.maps.Size(0, -26),
  content: ReactDOMServer.renderToString(
    <div className="p-2 text-xs">
      Stops ${overlappedStops.join(", ")}
    </div>,
  ),
});

infoWindow.open({ shouldFocus: false, map });
```

### Steps to Reproduce

1. Create a text input field.
2. Create a map.
3. Create a DIV element that, when hovered over, displays an InfoWindow on the map with shouldFocus set to false.
4. Focus on the text input field.
5. Hover over the DIV element.

**Actual Result**: The text input field loses focus.
**Expected Result**: The text input field should remain focused.

### Environment

- Library version: @vis.gl/react-google-maps@0.8.0
- Google maps version: weekly
- Browser and Version: Opera One(version: 108.0.5067.29)
- OS: macOS 13.6.5 (22G621)

**Comment by usefulthink** [comment]
This might have been the reason: 2f4b508a3da
The fix was just published with 0.8.1, can you check if the issue still exists after updating?

**Comment by ngothanhtai** [comment]
Hi @usefulthink, it works now, thanks.
